specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -4301,6 +4301,13 @@
           "type": "integer",
           "format": "int32",
           "description": "The scale unit for this vpn gateway."
+        },
+        "ipConfigurations": {
```

Comment:
Is this a read-only or read/write property?

Reply:
Can you mark it as read-only in the swagger?

---
In reply to: [438295999](https://github.com/Azure/azure-rest-api-specs/pull/9722#discussion_r438295999) [](ancestors = 438295999)

Reply:
@majastrz Done

Reply:
I don't see it.

---
In reply to: [439132364](https://github.com/Azure/azure-rest-api-specs/pull/9722#discussion_r439132364) [](ancestors = 439132364)

Reply:
@majastrz Moved to 2020-06 version

Reply:
I definitely see the move to the June API version. I mean I don't see the read-only part, which you said was "Done".